core/src/main/java/org/bitcoinj/wallet/KeyChainGroupStructure.java

```diff
@@ -108,6 +108,8 @@ static HDPath purpose(ScriptType scriptType) {
             return HDPath.BIP44_PARENT;
         } else if (scriptType == ScriptType.P2WPKH) {
             return HDPath.BIP84_PARENT;
+        } else if (scriptType == ScriptType.P2TR) {
```

Comment:
I think we should at the same time define a path for BIP32 – I'd pick `m/2'`.

Reply:
Yeah, that makes sense. We should also consider _not_ using the `ACCOUNT_ZERO`, `ACCOUNT_ONE` constants for these child numbers as they are misleading in the BIP32 context.